cpp/ql/lib/semmle/code/cpp/commons/Buffer.qll

```diff
@@ -1,5 +1,5 @@
 import cpp
-import semmle.code.cpp.dataflow.DataFlow
+private import semmle.code.cpp.ir.dataflow.DataFlow
```

Comment:
I made this private, as there's no guarantee that the user of this library uses the same dataflow library. Alternative would be to make a copy of the library for ir dataflow.

Reply:
Does this need a change note?

Reply:
Well spotted! Indeed, I don't think any library should publicly export such dependencies. This technically is a breaking change (although a very trivial one for users to fix if they forgot to import the dataflow library in their own queries), so we should probably add a change note.

Reply:
Done.